auth.c

```diff
@@ -53,13 +68,95 @@ void janus_auth_deinit(void) {
 	if(allowed_plugins != NULL)
 		g_hash_table_destroy(allowed_plugins);
 	allowed_plugins = NULL;
+	if(auth_secret != NULL)
+		g_free(auth_secret);
+	auth_secret = NULL;
 	janus_mutex_unlock(&mutex);
 }
 
+gboolean janus_auth_check_signature(const char *token, const char *realm) {
+	if (!auth_enabled || auth_secret == NULL)
+		return FALSE;
+	gchar *token_signature = strchr(token, ':') + 1;
+	gchar *token_metadata = g_strndup(token, token_signature - token - 1);
+	gchar **list = g_strsplit(token_metadata, ",", 3);
+	/* need at least an expiry timestamp and realm */
```

Comment:
As a convention (yeah I know, I'm boring :smile: ), all sentences in comments start with a capitalized letter (Need at least...), same for the comments that follow (and maybe others in the patch).

Reply:
I was expecting a few of these formatting things - I tried but absorbing code styles is hard 😋